plugin.cpp

```diff
@@ -21,6 +21,14 @@
 using namespace std;
 
 #define PLUGIN_NAME "Csv"
+#define CONFIG	"{\"plugin\" : { \"description\" : \"" PLUGIN_NAME " C south plugin\", " \
+			"\"type\" : \"string\", \"default\" : \"" PLUGIN_NAME "\", \"readonly\" : \"true\" }, " \
+		"\"asset\" : { \"description\" : \"Asset name\", " \
+			"\"type\" : \"string\", \"default\" : \"Vibration\", \"order\": \"1\", \"displayName\": \"Asset Name\", \"mandatory\": \"true\" }," \
+		"\"file\" : { \"description\" : \"Name of the file to read\", " \
+			"\"type\" : \"string\", \"default\" : \"\", \"order\": \"2\", \"displayName\": \"Path Of File\" }" \
+		"} "
+
```

Comment:
No need of this define `CONFIG`, as we moved this to `default_config` as with latest develop.